.github/workflows/netlify.yml

```diff
@@ -0,0 +1,83 @@
+name: "📤 Upload Preview Build to Netlify"
+on:
+  workflow_run:
+    workflows: ["🌐 Build the matrix.org website"]
+    types:
+      - completed
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: >
+      ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      # There's a 'download artifact' action but it hasn't been updated for the
+      # workflow_run action (https://github.com/actions/download-artifact/issues/60)
+      # so instead we get this mess:
```

Comment:
wouldn't it be easier just to put these steps in the other workflow?

Reply:
If we do that and for some reason netlify fails, then the build workflow status won’t be `success`.

Reply:
well yes, but how much is that really a problem?